### ISSUE TITLE
trtexec does not show qps for onnx model

### ISSUE THREAD

**Issue by robbyneven** [NVIDIA/TensorRT#1162]
## Description

When building a TensorRT engine with trtexec for an onnx model (exported from PyTorch with dynamic batch dimension), the inference benchmark results always show a qps of 0. Is this normal behaviour, or did I do something wrong with the dynamic shapes? I read that when parsing an onnx model, the batch size needs to be explicit. Therefore, I set the min, opt, and max shapes, and provided the inference shape as well. The timing results, however, seem to make sense. 

`trtexec --onnx=branchedUNet.onnx --saveEngine=branchedUNet.engine --workspace=10000 --minShapes=input:1x3x288x768 --optShapes=input:16x3x288x768 --maxShapes=input:16x3x288x768 --shapes=input:16x3x288x768 --fp16`

Also, when only loading the serialised engine with trtexec, and providing the inference shape, the qps is 0.

[03/31/2021-10:02:13] [I] Starting inference
[03/31/2021-10:02:16] [I] Warmup completed 0 queries over 200 ms
[03/31/2021-10:02:16] [I] Timing trace has 0 queries over 3.06459 s
[03/31/2021-10:02:16] [I] Trace averages of 10 runs:
[03/31/2021-10:02:16] [I] Average on 10 runs - GPU latency: 23.6466 ms - Host latency: 28.7329 ms (end to end 44.1367 ms, enqueue 0.342834 ms)
[03/31/2021-10:02:16] [I] Average on 10 runs - GPU latency: 23.7594 ms - Host latency: 28.866 ms (end to end 38.7267 ms, enqueue 0.338339 ms)
[03/31/2021-10:02:16] [I] Average on 10 runs - GPU latency: 23.6876 ms - Host latency: 28.7744 ms (end to end 47.2063 ms, enqueue 0.381293 ms)
[03/31/2021-10:02:16] [I] Average on 10 runs - GPU latency: 23.6638 ms - Host latency: 28.7282 ms (end to end 47.1287 ms, enqueue 0.347675 ms)
[03/31/2021-10:02:16] [I] Average on 10 runs - GPU latency: 23.8041 ms - Host latency: 28.8548 ms (end to end 47.4136 ms, enqueue 0.337329 ms)
[03/31/2021-10:02:16] [I] Average on 10 runs - GPU latency: 23.7949 ms - Host latency: 28.8748 ms (end to end 47.2182 ms, enqueue 0.343457 ms)
[03/31/2021-10:02:16] [I] Average on 10 runs - GPU latency: 23.6925 ms - Host latency: 28.8286 ms (end to end 47.2176 ms, enqueue 0.296838 ms)
[03/31/2021-10:02:16] [I] Average on 10 runs - GPU latency: 23.7572 ms - Host latency: 28.8111 ms (end to end 36.7125 ms, enqueue 0.30946 ms)
[03/31/2021-10:02:16] [I] Average on 10 runs - GPU latency: 23.706 ms - Host latency: 28.7839 ms (end to end 46.8106 ms, enqueue 0.311157 ms)
[03/31/2021-10:02:16] [I] Average on 10 runs - GPU latency: 23.7479 ms - Host latency: 28.8415 ms (end to end 47.3089 ms, enqueue 0.318481 ms)
[03/31/2021-10:02:16] [I] Average on 10 runs - GPU latency: 23.7103 ms - Host latency: 28.767 ms (end to end 47.2513 ms, enqueue 0.317187 ms)
[03/31/2021-10:02:16] [I] Average on 10 runs - GPU latency: 23.7358 ms - Host latency: 28.8057 ms (end to end 47.2588 ms, enqueue 0.369629 ms)
[03/31/2021-10:02:16] [I] Host Latency
[03/31/2021-10:02:16] [I] min: 28.6117 ms (end to end 28.8083 ms)
[03/31/2021-10:02:16] [I] max: 29.1868 ms (end to end 48.8433 ms)
[03/31/2021-10:02:16] [I] mean: 28.8083 ms (end to end 45.4638 ms)
[03/31/2021-10:02:16] [I] median: 28.8022 ms (end to end 47.1994 ms)
[03/31/2021-10:02:16] [I] percentile: 29.073 ms at 99% (end to end 48.7479 ms at 99%)
[03/31/2021-10:02:16] [I] throughput: 0 qps
[03/31/2021-10:02:16] [I] walltime: 3.06459 s
[03/31/2021-10:02:16] [I] Enqueue Time
[03/31/2021-10:02:16] [I] min: 0.258545 ms
[03/31/2021-10:02:16] [I] max: 0.594177 ms
[03/31/2021-10:02:16] [I] median: 0.296036 ms
[03/31/2021-10:02:16] [I] GPU Compute
[03/31/2021-10:02:16] [I] min: 23.5582 ms
[03/31/2021-10:02:16] [I] max: 23.896 ms
[03/31/2021-10:02:16] [I] mean: 23.7266 ms
[03/31/2021-10:02:16] [I] median: 23.7057 ms
[03/31/2021-10:02:16] [I] percentile: 23.8857 ms at 99%
[03/31/2021-10:02:16] [I] total compute time: 2.98955 s
&&&& PASSED TensorRT.trtexec # trtexec --onnx=branchedUNet.onnx --saveEngine=branchedUNet.engine --workspace=10000 --minShapes=input:1x3x288x768 --optShapes=input:16x3x288x768 --maxShapes=input:16x3x288x768 --shapes=input:16x3x288x768 --fp16

## Environment

**TensorRT Version**: 7.2
**NVIDIA GPU**: Tesla V100
**NVIDIA Driver Version**: 450.80.02
**CUDA Version**: 11.2
**CUDNN Version**: 
**Operating System**: 
**Python Version (if applicable)**: 3.8
**Tensorflow Version (if applicable)**: 
**PyTorch Version (if applicable)**: 1.8
**Baremetal or Container (if so, version)**: nvcr.io/nvidia/tensorrt:21.03-py3

**Comment by ShawnNew** [comment]
same problem, any solution? @rmccorm4

**Comment by ttyio** [comment]
Hello @robbyneven , could you try 8.0EA from https://developer.nvidia.com/nvidia-tensorrt-download, thanks!

**Comment by ttyio** [comment]
Close since no activity for more than 3 weeks, please reopen if you still have question, thanks!
